### ISSUE TITLE
Translation is coming.

### ISSUE THREAD

**Issue by SrGeneroso** [sveltejs/svelte.dev#588]
I know is a bit early, but since the [svelte.dev](svelte.dev) site has being rebuilt and the [sv cli](https://www.youtube.com/watch?v=fAPFsRP-mbc&t=11700s) now has internationalization in scope I guess is time for svelte to become international.
I've asked before in different discord channels and seems like there are already things in motion, French community is the most involved by my experience. So let's get it done! It seems paraglide is the chosen one by the cli but we need to be implemented from the top.
I would love to help in making svelte more _esbelto_ and I've lurk around enough discord international communities to know that they are ready. Make the first implementation and we'll do the rest.

Let's the conquest of the world begin!

**Comment by dummdidumm** [comment]
The way we currently think about translating this repository is to make it easy to create a fork of this repository which you can translate, and to provide some tooling through which you can see what changes have been made to the documentation since a given commit. We may also add ways to make it easier to translate the text within components. As a result, you would merge all changes from the original repository into your fork except the changes to the documentation, which you would discard and instead translate.

**Comment by SrGeneroso** [comment]
I hear a rumble on the french society discord, I'm so excited! I think it would be awesome to have a single domain with all the languages so translators don't have to think about hosting and domain ownership, nonetheless is still early and forking seem to have worked in the past so let's begin with that.
I'll wait a couple of weeks to let the omnisite (🧑‍🍳😘) to settle, in the meanwhile I'll be checking the different international society channels to feel the interest and follow up with it.
As some efforts are going to be made to facilitate the forking process please keep considering to have translations on the main site. I18n is a real challenge and a real concern for accessibility, maybe svelte core will find an elegant solution that paves the way for a better www.

**Comment by SrGeneroso** [comment]
Btw, I know maybe is too much to ask, but... would svelte consider allowing subdomains for the translated sites? In the [french svelte society](https://discord.gg/axFFuzWe) discord are [discussing ](https://discord.com/channels/1065973573480632360/1072140419535745064/1298718373076537365) renaming the repo to align better with their [sveltefr](https://kit.sveltefr.dev/) domain. I noted that it would be really cool to have fr.svelte.dev as their domain. Please let me know if this could be achieved and what would take to make it possible. I'll be soon poking some eyes on the spanish counterpart and would be nice to know how could be achieved.

**Comment by bleucitron** [comment]
> please keep considering to have translations on the main site

From what i understand, all content to be translated is already present in this repo:
- markdown files in in https://github.com/sveltejs/svelte.dev/tree/main/apps/svelte.dev/content
- UI elements in https://github.com/sveltejs/svelte.dev/tree/main/packages
- and probably other stuff in other folders

**Comment by SrGeneroso** [comment]
> please keep considering to have translations on the main site

What I'm asking is that in the future (when translations achieve more quality and automatization), they bring the translation into the main site and properly implement it. Forks and externally deployed sites are a good solution but I dream of a day where svelte.dev is for all.

Mainly I'm subtly trying to make core team work their magic with i18n and maybe consider having it as a main feature for ¿svelte 6?.

In the meantime. Is there a conversation already in github or a discord thread about the translation effort that I should be aware of?
Do different languages communities have a common space already to share their finds and processes?

 I've seen some [talks](https://discord.com/channels/457912077277855764/750401468569354431/1298711552035131492) spread on the main discord contributing channel, but it would be nice to have a dedicated one.
My reason is that for example I've seen you've started to work in some [github actions](https://github.com/bleucitron/svelte.dev/actions/runs/11524741174/workflow) to track changes on the omnisite docs, and that's the kind of stuff we could share regardless of the language. 
Or the [conversation ](https://discord.com/channels/457912077277855764/750401468569354431/1298711552035131492) about how some parts of the site would be broken (the playground supabase db). Really insightful thread that goes missing in the general noise.
I feel that translation is going to involve a lot of effort and it is happening in parallel in the different international society servers. Some discussion would greatly benefit if the happen publicly in the main repo of svelte and some would move faster if we have a dedicated channel in the discord contributors section.
Let me know what you think.

**Comment by bleucitron** [comment]
> some would move faster if we have a dedicated channel in the discord contributors section.

A dedicated channel in Contributors on the official Discord might be very relevant indeed.

**Comment by bleucitron** [comment]
This was recently closed, and reopened.

Does this mean the core team has some intentions towards translations ?

I'm currently about 80% through [translating the docs into french](https://github.com/bleucitron/svelte.dev), which are currently available [here](sveltefr.dev), and would love to see it integrated somehow into the main site, should you see it fit.
